docs/code/GraphRunner.md

```diff
@@ -0,0 +1,123 @@
+﻿# JSON Graph format
+
+The entry point graph in TLC is an array of _nodes_. Each node is an object with the following fields:
+
+- _name_: string. Required. Name of the entry point.
+- _inputs_: object. Optional. Specifies non-default inputs to the entry point. 
+Note that if the entry point has required inputs (which is very common), the _inputs_ field is requred.
+- _outputs_: object. Optional. Specifies the variables that will hold the node's outputs.
+
+## Input and output types
+The following types are supported in JSON graphs:
+
+- _string_. Represented as a JSON string, maps to a C# string.
```

Comment:
>string [](start = 55, length = 6)

Should these types when listed here be listed as `string` vs. plain old string, since we are using C# keywords to describe them? (E.g.: `string`, `float`, `double`, `bool`, `enum`, etc.)